addons/mail/static/tests/composer/composer_tests.js

```diff
@@ -841,11 +841,50 @@ QUnit.test(
 
 QUnit.test("Show a thread name in the recipient status text.", async (assert) => {
     const pyEnv = await startServer();
-    const partnerId = pyEnv["res.partner"].create({ name: "test name" });
+    const partnerId = pyEnv["res.partner"].create({ name: "test name", email: "test@odoo.com" });
+    pyEnv["mail.followers"].create({
+        is_active: true,
+        partner_id: partnerId,
+        res_id: partnerId,
+        res_model: "res.partner",
+    });
     const { openFormView } = await start();
     await openFormView("res.partner", partnerId);
     await click("button:contains(Send message)");
-    assert.containsOnce($, '.o-mail-Chatter:contains(To: Followers of "test name")');
+    assert.containsOnce($, ".o-mail-Chatter:contains(To: test.)");
+    assert.containsOnce($, 'span[title="test@odoo.com"]');
+});
+
+QUnit.test("Show follower list when there is more than 5 followers.", async (assert) => {
+    const pyEnv = await startServer();
+    const partnerIds = pyEnv["res.partner"].create([
```

Comment:
can be in a single for loop if wants even more clean code

Reply:
I know, but not sure it better, It's just 6 elements